spec/support/helpers.rb

```diff
@@ -1,4 +1,4 @@
-require 'support/helpers/session_helpers'
+require 'support/helpers/features_helpers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.